compiler/rustc_hir_analysis/src/hir_ty_lowering/mod.rs

```diff
@@ -2265,18 +2265,11 @@ impl<'tcx> dyn HirTyLowerer<'tcx> + '_ {
         if let Some(lit_input) = lit_input {
```

Comment:
nit: can you change this to avoid explicit returns? e.g. by just using a `lit_input.filter(has_aliases).map(lit_to_const)`